habitat/tasks/nav/nav.py

```diff
@@ -514,6 +521,118 @@ def get_observation(
         )
 
 
+@registry.register_sensor(name="SemanticCategorySensor")
+class SemanticCategorySensor(Sensor):
```

Comment:
Do we use this sensor for training or only for visualization purposes?

Reply:
This sensor is used for training models with GT semantic goal inputs.

Reply:
@srama2512, yes. But that GT data isn't available during evaluation and from this PR isn't clear how it will be replaced.